routes/web.php

```diff
@@ -14,9 +15,12 @@
 */
 
 Route::get('/', function () {
-    return view('welcome');
+    return view('dashboard');
 });
 
+Route::get('/home', [PostController::class, 'index']);
+Route::post('/home', [PostController::class, 'store']);
```

Comment:
```suggestion
Route::post('/home', [PostController::class, 'store']);
```

You can apply auth middleware here to enforce the user is authenticated, allowing you to get the user_id from `$request->user()->id` rather than trusting user input.